### PR TITLE
Translated "Update _index.ko.md" from English to Korean

### PR DESCRIPTION
**Pull request by joft-ware**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Translated "Update _index.ko.md" from English to Korean

### Motivation and Context

Missing translation

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by diemol** [comment]
Thank you, @joft-ware!
Could you please sign the CLA so I can merge this right away?

**Comment by joft-ware** [comment]
Sure! I just signed

File: docs_source_files/content/remote_webdriver/_index.ko.md [hunk 1]
```diff
@@ -4,18 +4,10 @@ chapter: true
...
-configured so that it can run your tests on a separate machine.
+로컬 환경에서와 같은 방법으로 WebDriver를 원격으로 사용할 수도 있습니다.
+중요한 차이점은 다른 장치에서 작업을 수행하기 위해 원격 WebDriver가 구성되어있어야 한다는 점입니다.
 
-A remote WebDriver is composed of two pieces: a client and a
-server. The client is your WebDriver test and the server is simply a
-Java servlet, which can be hosted in any modern JEE app server.
\ No newline at end of file
+원격 WebDrive는 두 가지 요소로 이루어져 있습니다: 클라이언트 그리고 서버.
+클라이언트는 당신의 WebDriver teset이며 서버는 간단한 Java servelet의 일종이고 JEE app 서버에 호스팅될 수 있습니다.
```
Comment (harsha509) [inline docs_source_files/content/remote_webdriver/_index.ko.md]: Servlet

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @joft-ware,

Thank you for the PR. Can you please look into the suggestions.

Thanks!

File: docs_source_files/content/remote_webdriver/_index.ko.md (hunk 1 above)
Comment (joft-ware) [reply docs_source_files/content/remote_webdriver/_index.ko.md]: Sorry I just fixed it

**Review by harsha509 — APPROVED** [review]
Thank you @joft-ware!

**Pull request merged**